viewer/cypress/vectortile-view-test.cy.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpClientModule } from '@angular/common/http'
+import { LoggerModule } from 'ngx-logger'
+import { environment } from 'src/environments/environment'
+import { VectortileViewComponent } from 'src/app/vectortile-view/vectortile-view.component'
+
+describe('Vectortiled-view-test.cy.ts', () => {
+  it.skip('Skipped unable to supply vectortile as feature yet, schould Write File Test', function () {
+    cy.request('GET', 'https://api.pdok.nl/kadaster/bestuurlijkegebieden/ogc/v1_0-preprod/tiles/NetherlandsRDNewQuad/2/1/1?f=mvt').then(

```

Comment:
Kan je hier (en eigenlijk in alle testen) dummy url's gebruiken? Aangezien de boel toch gemocked wordt. We willen preprod url's niet adverteren. Zou iets als `https://api.pdok.example/etc/etc/` gebruiken. De `.example` tld is speciaal voor deze doeleinden bedoeld.